source/releasenotes/about.rst

```diff
@@ -16,18 +16,70 @@
+
+Some of the changes are presented in this quick summary (this is not a complete list):
+
+* Allow users of all types to create L2 networks
+* systemd: Fix -Dpid arg passing to systemd usage service
+* Add back ability to disable backup of snapshot to
+* keystore: don't restart systemvm cloud.service post cert
+* Keep iotune section in the VM's XML after live migration
+* Copy template to target KVM host if needed when migrating local <> local storage
+* systemd: fix services to allow TLS configurations via
+* java.security.ciphers
+* feature: add libvirt / qemu io bursting
+* Add back ability to disable backup of snapshot to secondary
+* API: add command to list management servers
+* UI: Fix "Migrate instance to another host" popup modal
+* UI: Fix issue with verification of ipv4/ipv6 address.
+* UI: Fix UI bug: Create Network Offering Popup has no fields
+* UI: Update jquery and related libraries
+* network: Offerings do not have to have Security Grouping enabled
+* Vmware offline migration
+* IP address acquired with associate ip address is marked as source nat
+* ipv6: allow Secondary IPv6 addresses to be EUI-64
+* ipv6: Send userdata to Virtual Router if IPv6 is enabled
+* ipv6: Calculate IPv6 address instead of fetching one from a pool
+* ipv6: Advanced Networking Security Groups are supported
+* ipv6: Allow specification of IPv6 details when creating Basic network
+* Enable remote debugging for the management server, usage server, and KVM agent
+* Security Group: add secondary ips to the correct ipset based on ip family (4.11)
+* Security Group: add secondary ips to the correct ipset
+* XenServer: fix Security Groups 'vmops' script
+* XenServer: Support online storage migration from non-managed to managed storage
+* KVM: Security Group enhancements and refactor old code
+* KVM: Properly report available memory to Management Server
+* KVM: Add influxdb to statscollector
+* KVM: Refactory VXLAN script and add IPv6 support
+* KVM: Set amount of queues for Virtio SCSI driver to vCPU of Instance
+* KVM: Use 'ip route show default 0.0.0.0/0' to find the default gateway
+* KVM: Add KVM Guest OS mapping for Windows Server 2019
+* KVM: Enable DPDK support on KVM
+* Prevent corner case for infinite PrepareForMaintenance
+* Enhance bypass vlan overlap checkAllow KVM VM live migration with ROOT volume on file storage type
+* Destroy VM also removes volumes
+* Users are able to change/edit the protocol of an ACL rule
+* allows to remove local primary storage
+* Allow password enabled for iso (#2745)
+* Support requesting a specific IPv4 address in Basic Networking during Instance creation
+* Adding zone disablement during deletion of the range
+* Display mac address in nic detail view
+* Cleanup methods, classes, and POMs
+
 
 What's New in 4.11.2.0
 ----------------------
 
-The new |release| version is a |version| maintainance release containing over 70
```

Comment:
|release| and |version| are a global variables from conf.py they should not be hardcoded

Reply:
This is for 4.12.0.0, when we move the next release; we move the variable sentence to the above/latest release.